src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { AngularFireModule } from "angularfire2";
+
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { AngularFireDatabaseModule } from "angularfire2/database";
+import { AngularFireAuthModule } from "angularfire2/auth";
+import { AuthService } from "../services/auth.service";
+import { AuthComponent } from "./auth.component";
+import { AuthGuard } from "../services/auth-guard.service";
+import { MaterialComponentsModule } from "../material/material.module";
+import { config } from "../../../firebase.config";
```

Comment:
pls, do not mix modules, components and services. it is hard to read. Group them and separate with extra space.